build-woocommerce

```diff
@@ -5,3 +5,11 @@ set -e
 . ./build
 
 build_plugin "woocommerce" "wp-content/plugins/indodana-payment/library/Indodana/Payment"
+
+# Release(WooCommerce): move contents of indodana-payment
+# directly under plugin dir before zipped
+TARGET_DIR=".build/woocommerce"
```

Comment:
I'll accept for now, but you might consider changing 1st parameter of `build_plugin` function to use `TARGET_DIR` instead of `PLUGIN_NAME`

Reply:
This will become more prevalent when you start adjust other merchant plugin

Reply:
nvm, you need it on `SOURCE_NAME`